qa/tasks/cephfs/test_data_scan.py

```diff
@@ -447,9 +447,9 @@ def get_state(mds_id):
         if False:
             with self.assertRaises(CommandFailedError):
                 # Normal reset should fail when no objects are present, we'll use --force instead
-                self.fs.journal_tool(["journal", "reset"], 0)
+                self.fs.journal_tool(["journal", "reset", "--yes-i-really-really-mean-it"], 0)
 
-        self.fs.journal_tool(["journal", "reset", "--force"], 0)
+        self.fs.journal_tool(["journal", "reset", "--force", "--yes-i-really-really-mean-it"], 0)
```

Comment:
it seems journal reset is widely used in test cases, how about creating a new func `journal_reset()`?

Reply:
@dparmar18 
Could you please create a tracker for that (and assign to me), so that this `urgent` patch could make progress on it's own ?